docs_src/environment_variables.md

```diff
@@ -0,0 +1,20 @@
+When running docker-run.sh script, we support environment variables as input for containers. Here is a list of environment variables and how you can apply it to docker-run.sh script as input
+
```

Comment:
this is duplicated messages as in pipelinerun.md please remove it

Reply:
This is a description to environment variables, I know that I duplicated messages, but I think it is helpful when user just open this file and to understand what it is.

Reply:
please re-phrase it to reduce the duplicate messages if you feel compelling to have one